replica/database.cc

```diff
@@ -2011,7 +2011,7 @@ max_purgeable memtable_list::get_max_purgeable(const dht::decorated_key& dk, is_
 }
```

Comment:
In patch comment, it's worth adding a reference to whatever pr/commit made this "now that it's safe to flush an empry memtable list"